src/main/java/org/opentripplanner/transit/service/TransitEditorService.java

```diff
@@ -40,4 +40,8 @@ void addTripOnServiceDateForTripAndDay(
   FeedScopedId getOrCreateServiceIdForDate(LocalDate serviceDate);
 
   void setTransitLayer(TransitLayer transitLayer);
+
+  boolean hasRealtimeTransitLayer();
+
+  void setRealtimeTransitLayer(TransitLayer realtimeTransitLayer);
```

Comment:
This need documentation. Which part of the code is expected to set this?

Reply:
done